src/samples/custom-roles.json

```diff
@@ -0,0 +1,568 @@
+{
+  "schema_version": "1.1",
+  "type_definitions": [
+    {
+      "type": "user",
+      "relations": {}
+    },
+    {
+      "type": "team",
+      "relations": {
+        "member": {
+          "this": {}
+        }
+      },
+      "metadata": {
+        "relations": {
+          "member": {
+            "directly_related_user_types": [
+              {
+                "type": "user"
```

Comment:
can `team#member` be assigned here?

Reply:
Teams are not nested in the Custom Roles sample, but they can be made to be - this is just following the samples [here](https://github.com/openfga/sample-stores/tree/main/stores/custom-roles).

Ideally they wouldn't be copied over like this (git submodule or npm import), but we'll tackle that in the future.